Userland/Libraries/LibArchive/TarStream.h

```diff
@@ -34,6 +34,10 @@ class TarFileStream : public InputStream {
 class TarInputStream {
 public:
     TarInputStream(InputStream&);
+    ~TarInputStream()
+    {
+        m_stream.handle_any_error();
```

Comment:
As far as I can tell, we shouldn't just clear the error flags on destruction. If those are still set here, that means that some earlier operation on the stream failed and we haven't handled that issue right then and there when it appeared (which is also what I think that `VERIFY` is supposed to guard against).